migrator/migrations/m_192_to_m_193_report_config_migrations/migration_impl.go

```diff
@@ -0,0 +1,172 @@
+package m192tom193
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	updatedSchema "github.com/stackrox/rox/migrator/migrations/m_192_to_m_193_report_config_migrations/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/pkg/uuid"
+	"gorm.io/gorm/clause"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+func migrate(database *types.Databases) error {
+	db := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, db, updatedSchema.CreateTableReportConfigurationsStmt)
+	db = db.WithContext(database.DBCtx).Table("report_configurations")
```

Comment:
This has the issue that Mandar ran into.  If you were to do another migration on this table, it would fail because of the statement cache.  Take a look at https://github.com/stackrox/stackrox/blob/9291db7ba40a387c13947585d33e32155d4a3656/migrator/migrations/m_191_to_m_192_vulnerability_requests_searchable_scope/migration_impl.go#L31.

you will need another handle for querying and specifically select just the data you are interested in.  Alternatively you could probably add `.Select` and list every column specifically if you didn't want to have a specific query handle.